src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/AttachedCollectionItemBase.cs

```diff
@@ -42,7 +42,8 @@ public abstract class AttachedCollectionItemBase
         private static readonly HashSet<Type> s_supportedPatterns = new HashSet<Type>
         {
             typeof(ITreeDisplayItem),
-            typeof(IBrowsablePattern)
+            typeof(IBrowsablePattern),
+            typeof(IContextMenuPattern)
```

Comment:
Should it return null for this pattern if ContextMenuController is null?

Reply:
It works out the same in the end: http://index/?leftProject=Microsoft.VisualStudio.DesignTools.Diagnostics&leftSymbol=kx9opq4zj20s&file=Tool%5CUI%5CControls%5CTreeView%5CInteractionPatternProvider.cs&line=30